x/dualstaking/keeper/delegator_reward.go

```diff
@@ -113,12 +116,12 @@ func (k Keeper) CalcEffectiveDelegationsAndStake(stakeEntry epochstoragetypes.St
 
 // CalcDelegatorReward calculates a single delegator reward according to its delegation
 // delegatorReward = delegatorsReward * (delegatorStake / totalDelegations) = (delegatorsReward * delegatorStake) / totalDelegations
-func (k Keeper) CalcDelegatorReward(delegatorsReward math.Int, totalDelegations math.Int, delegation types.Delegation) math.Int {
+func (k Keeper) CalcDelegatorReward(ctx sdk.Context, delegatorsReward sdk.Coins, totalDelegations math.Int, delegation types.Delegation) sdk.Coins {
 	// Sanity check - totalDelegations != 0
 	if totalDelegations.IsZero() {
-		return math.ZeroInt()
+		return sdk.NewCoins(sdk.NewCoin(k.stakingKeeper.BondDenom(ctx), math.ZeroInt()))
```

Comment:
cant we return here empty sdk.coins?

Reply:
done. see https://github.com/lavanet/lava/commit/42ab2dfe1d44e07f5a6ce2bbe3904fc7b1638c1f